util/cmake/FindSeqAn.cmake

```diff
@@ -112,54 +112,51 @@ endif ()
 
 # Recognize Clang compiler.
 
-set (COMPILER_IS_CLANG FALSE)
+set (COMPILER_CLANG FALSE)
+set (COMPILER_GNU FALSE)
```

Comment:
In the source code we call it COMPILER_GCC so maybe we should do that here, as well!